src/containers/my-cooperations/cooperation-completion/CooperationCompletion.tsx

```diff
@@ -25,13 +46,28 @@ const CooperationCompletion = () => {
         title={t('cooperationsPage.cooperationDetails.closeCooperationTitle')}
       >
         <AppButton
+          data-testid='close-cooperation-btn'
```

Comment:
Probably you should disable this button when cooperation is no active (already closed, or in closing process)
```suggestion
          data-testid='close-cooperation-btn'
          disabled={cooperation.status!=='active'}
```